Source/Scene/Cesium3DTileset.js

```diff
@@ -1617,6 +1634,34 @@ define([
 
         // Number of commands added by each update above
         tileset._statistics.numberOfCommands = (commandList.length - numberOfInitialCommands);
+
+        if (tileset.debugShowGeometricError) {
+            tileset._geometricErrorLabels.removeAll();
+            for (i = 0; i < length; ++i) {
+                tile = selectedTiles[i];
+                if (tile.selected) {
+                    var boundingVolume = tile._boundingVolume.boundingVolume;
+                    var halfAxes = boundingVolume.halfAxes;
```

Comment:
Yes, the `boundingVolume` property of a `TileBoundingRegion` is also an `OrientedBoundingBox`